modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Setting the content with an attribute that contains a self-closing HTML tag did not preserve the tag. #TINY-10088
 - Resize handles would not appear on editable images in a non-editable context. #TINY-10118
+- Indent/Undent a list that contains a not `li` with a `ul`/`ol` as first element make disappear all others elements inside the `li`. #TINY-10213
```

Comment:
Toggling a list that contains an LI element having another list as its first child would remove the remaining content within that LI element.

Reply:
changed